Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:14.3.0-alpine as builder
+
+WORKDIR /opt/auth-service
+
+RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool autoconf automake \
+    && cd $(npm root -g)/npm \
+    && npm config set unsafe-perm true \
+    && npm install -g node-gyp 
+
+# check in .dockerignore what is skipped during copy
+COPY . .
+RUN npm ci
+
+# Create empty log file & 
+RUN mkdir ./logs && touch ./logs/combined.log
+
+# link stdout to the application log file
+RUN ln -sf /dev/stdout ./logs/combined.log
+
+# USER node 
+# copy bundle
+# COPY --chown=node --from=builder /opt/auth-service/ .
+
+# cleanup
+RUN apk del build-dependencies
+# RUN npm prune --production
+
+RUN npm run test:unit
+EXPOSE 4004
+CMD ["npm", "run", "start"]
```

Comment:
This `Dockerfile` is fine for now, although it would be better to have the unit tests as part of the base image, so we don't need to put the test sources and `devDependencies` in the final image.

Reply:
its MVP Docker file, I put test:unit there to verify everything needed is in container ;-)

Reply:
of course, there will be changes in future